homeassistant/components/home_connect/common.py

```diff
@@ -7,26 +7,31 @@
 
 from aiohomeconnect.model import EventKey
 
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import entity_registry as er
```

Comment:
To improve readability, I suggest adding the import `from homeassistant.helpers.entity_registry import EntityRegistry` as well and use it for typing purposes instead of `er.EntityRegistry`. It would be more consistent with the already existent code.